app/src/main/java/net/bonysoft/magicmirror/MainActivity.java

```diff
@@ -137,4 +155,40 @@ private DecimalFormat getTemperatureFormat() {
         temperatureFormat.setRoundingMode(RoundingMode.HALF_DOWN);
         return temperatureFormat;
     }
+
+    private final TwitterModule.TwitterListener tweetListener = new TwitterModule.TwitterListener() {
+
+        @Override
+        public void onNextTweet(final Status tweet) {
+            runOnUiThread(new Runnable() {
```

Comment:
I'd prefer having this already be called on the main thread instead of expecting the user to figure it out the hard way.